test/integration/globals/app-nav.js

```diff
@@ -46,18 +40,55 @@ context('App Nav', function() {
 
         expect(storageItem).to.be.false;
       });
-    cy
-      .getRadio(Radio => {
-        logoutStub = cy.stub();
-        Radio.reply('auth', 'logout', logoutStub);
-      });
 
     cy
       .get('.app-nav__header')
-      .should('contain', 'Cypress Clinic')
+      .should('contain', 'Workspace One')
       .should('contain', 'Clinician McTester')
       .as('mainNav');
 
+    cy
+      .get('@mainNav')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-workspace-name')
+      .should('contain', 'Cypress Clinic');
+
+    cy
+      .get('.picklist')
+      .find('.picklist__group')
+      .find('.picklist__item')
+      .first()
+      .should('contain', 'Workspace One')
+      .should('have.class', 'is-selected')
+      .next()
+      .should('contain', 'Workspace Two')
+      .should('not.have.class', 'is-selected');
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-bottom')
+      .contains('Help & Support')
+      .should('have.attr', 'href')
+      .and('contain', 'help.roundingwell.com');
+
+    cy
+      .get('.picklist')
+      .find('.app-nav__picklist-bottom')
+      .contains('Sign Out')
+      .should('have.attr', 'href')
+      .and('contain', '/logout');
```

Comment:
Moved these up to the top of the test. I think the `Sign Out` code was originally placed at the bottom because it was navigating the test to a new page...?

And I also added some checks to ensure the organization name, workspaces, and help/sign-out items are displayed.